docs/tutorials/advanced/classification_of_strategies.rst

```diff
@@ -43,7 +43,7 @@ strategy's logic as default, we use :code:`Classifiers[<classifier>](
 <strategy>)`::
 
     >>> from axelrod import Classifiers
-    >>> Classifiers['memory_depth'](axl.TitForTat)
+    >>> Classifiers['memory_depth'](axl.TitForTat())
```

Comment:
Why did this need to be changed?

Reply:
I think it's not necessary per se but there is a warning:
```
/home/user/repos/axelrod/Axelrod/axelrod/classifier.py:196: UserWarning: Classifiers are intended to run on player instances. Passed player Tit For Tat was initialized with default arguments.
```

Reply:
Makes sense.